config/routes.rb

```diff
@@ -49,9 +48,10 @@
   patch "/profile/:id/edit", to: "profile#update"
   patch "/profile/:id/password", to: "passwords#update"
 
-  namespace :user do
+  namespace :profile do
     get "/", to: "profile#index"
```

Comment:
Good catch here - I think this will enable us to be consistent with user stories.